Vaadin/capstone-2024/src/main/java/com/example/application/views/collegebasketballprediction/CollegeBasketballPredictionsView.java

```diff
@@ -85,18 +85,18 @@ public CollegeBasketballPredictionsView() {
 
 
         // action listeners for date, conference, and team search
-        datePicker.addValueChangeListener(event -> updateGridData(event.getValue(), comboBox.getValue().toString()));
-        comboBox.addValueChangeListener(event -> updateGridData(datePicker.getValue(), event.getValue().toString()));
-        //textField.addValueChangeListener(event -> updateGridData(datePicker.getValue(), event.getValue().toString(), textField.getValue().toString()));
-
+        datePicker.addValueChangeListener(event -> updateGridData(event.getValue(), comboBox.getValue().toString(), textField.getValue().toString()));
```

Comment:
Since you're reusing comboBox.getValue().toString() and textField.getValue().toString() many times, might be useful to just put each into a variable. Less typing, less verbose then.